clients/vault/resources/config/testnet/stellar_relay_config_sdftest2.json

```diff
@@ -5,10 +5,10 @@
   },
   "node_info": {
     "ledger_version": 19,
-    "overlay_version": 27,
-    "overlay_min_version": 24,
-    "version_str": "stellar-core 19.9.0.rc1 (064a2787acb9e98c70567523785333581ee1ffa4)",
+    "overlay_version": 28,
+    "overlay_min_version": 27,
+    "version_str": "stellar-core 19.12.0.rc2 (2109a168a895349f87b502ae3d182380b378fa47)",
     "is_pub_net": false
   },
-  "stellar_history_base_url": "http://history.stellar.org/prd/core-live/core_live_002/"
+  "stellar_history_base_url": ""
```

Comment:
Are we setting `stellar_history_base_url` as an empty string for all the nodes that are no longer in the network?

Reply:
it's only empty for testnet, since there's no archives on testnet. This is basically a `todo`, to make `stellar_history_base_url` field optional.

Reply:
I'm changing this slightly in [this](https://github.com/pendulum-chain/spacewalk/pull/366/files#diff-b50844933a930fbf26d7b10be28be063386b05d5993a1ca76a7cb1f0fc7be020) PR.